validator/accounts/v1/status.go

```diff
@@ -13,6 +13,8 @@ import (
 	"go.opencensus.io/trace"
 )
 
+const statusTimeout = 30 * time.Second /* Cancel if running over thirty seconds. */
```

Comment:
```suggestion
// statusTimeout defines a period after which request to fetch account statuses is cancelled.
const statusTimeout = 30 * time.Second
```

Consider updating comment.